game/src/debug/mod.rs

```diff
@@ -730,7 +730,8 @@ impl ContextualActions for Actions {
                 let center = if pts[0] == *pts.last().unwrap() {
                     // TODO The center looks really wrong for Volunteer Park and others, but I
                     // think it's because they have many points along some edges.
-                    Pt2D::center(&pts.iter().skip(1).cloned().collect())
+                    let v: Vec<_> = pts.iter().skip(1).cloned().collect();
```

Comment:
I could get rid of the intermediate value at the cost of a turbofish.

Reply:
> Actually yeah, could you please? Most of the places with all of the intermediate vars are a bit less readable.

Done.

> It's unfortunate that we have to collect at all, but consuming the iterator directly would take way more type declarations at the method definitions than is worth it.

Now that we have `impl Trait`, the type declarations might actually not so bad.

Reply:
Thanks! I'll keep `impl trait` in mind, but build times are definitely a problem already, so... [turbofish](https://turbo.fish/) is fine.